datahub/scripts/add-header-webcomponent.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 DATAHUB_DIR=${1:-/usr/share/nginx/html/datahub}
-SNIPPET="<script src='https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js'></script><geor-header></geor-header>"
+SNIPPET="<script src='https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js'></script><geor-header active-app='datahub' style='height:90px'></geor-header>"
```

Comment:
Would be nice if there was a way we could use the values from the `default.properties` here, especially for the `headerScript`.

Reply:
Then maybe this should be supported natively by datahub. Datahub having the functionality of reading a properties file in order to add a "header".

Reply:
For now we won't go that way, the "header" feature is still very much a geOrchestra thing. Let's see if other users need this as well, then we can revisit.

Reply:
Yep, I guess for now we don't have an alternative than to repeat some values from the `default.properties`.